### PR TITLE
Changes/Fixes

### PR DESCRIPTION
**Pull request by Metroid55**
-fixed golfclub description
-fixed gauntlet description
-fixed snap title
-added purpose text to anonymous mask
-added more words to ponify and hopefully fixed all uppercase for non ponified words 
-added more emotes to owoifier
-added 'givepointsrandom' chat command (please review if I did this correctly, because I don't have access to the givepoints lua file)

### CODE REVIEW

**Comment by Metroid55** [comment]
Any updates?

File: lua/swamp/chat/sv_commands/bounties.lua [hunk 1]
```diff
@@ -177,3 +177,42 @@ end, {
     global = false,
     throttle = false
 })
+
+--Could be it's own file, but eh
+RegisterChatCommand({'givepointsrandom', 'randomgivepoints'}, function(ply, arg)
+    local p = tonumber(arg)
```
Comment (swampservers) [inline lua/swamp/chat/sv_commands/bounties.lua]: p = math.floor(tonumber(arg) or 0)

Then check if p>0

Comment (Metroid55) [reply lua/swamp/chat/sv_commands/bounties.lua]: done

File: lua/swamp/chat/sv_commands/bounties.lua [hunk 2]
```diff
@@ -177,3 +177,42 @@ end, {
...
+        
+        ply:TryTakePoints(p, function()
+            if not IsValid(ply) then return end
+
+            if IsValid(rcvr) then
+                rcvr:GivePoints(p, function()
+                    ply:Notify("You gave " .. rcvr:Nick() .. " " .. p .. " points.")
+                    rcvr:Notify(ply:Nick() .. " gave you " .. p .. " of their points.")
+                end,
+                function() return end)
```
Comment (swampservers) [inline lua/swamp/chat/sv_commands/bounties.lua:206]: you don't need a fail callback if its empty, this just pollutes the code. you don't need to use a callback at all here, just do GivePoints(p)

**Pull request merged**